servicemanager/servicemanager.go

```diff
@@ -163,24 +144,15 @@ func New(
 		return nil, aoserrors.Wrap(err)
 	}
 
-	services, err := sm.serviceInfoProvider.GetLatestVersionServices()
+	services, err := sm.serviceInfoProvider.GetServices()
 	if err != nil {
 		return nil, aoserrors.Wrap(err)
 	}
 
 	for _, service := range services {
-		if err = sm.gidPool.AddID(service.GID); err != nil {
-			log.Errorf("Can't add service GID to pool: %v", err)
-		}
-
 		if service.Cached {
-			size, err := sm.getServiceSize(service.ServiceID)
-			if err != nil {
-				return nil, aoserrors.Wrap(err)
-			}
-
-			if err = sm.serviceAllocator.AddOutdatedItem(
-				service.ServiceID, size, service.Timestamp); err != nil {
+			if err := sm.serviceAllocator.AddOutdatedItem(
```

Comment:
Looks like long line (more than 120).
if !cached -> continue? To decrease nested levels.

Reply:
Done. 
line less than 120 characters